data/docs/example.mdx

```diff
@@ -63,41 +71,14 @@ const Panel = ({ title, children, wide }: Props) => (
 export default Panel;
 ```
 
-## panel/styles.tsx
-
-This is the styles file. It contains the CSS Components that are used in the main component file. As you can see, CSS Components are simply a way to map CSS classes to React component in a logical way, nothing more.
-
-```tsx
-import { styled } from "@phantomstudios/css-components";
-
-import css from "./styles.module.css";
-
-export const PanelWrapper = styled("div", {
-  css: css.PanelWrapper,
-  variants: {
-    wide: {
-      true: css.PanelWrapper_wide,
-    },
-  },
-});
-
-export const Title = styled("h2", {
-  css: css.Title,
-});
-
-export const Content = styled("section", {
-  css: css.Content,
-});
-```
-
-Please note that while this example employs standard CSS, you have the freedom to utilize any styling solution of your choice, such as SCSS.
+## styles.module.css
 
-## panel/styles.module.css
+Next is your bog standard CSS file. It contains all the CSS classes used by CSS Components. You're free to name them how you want, [but we do have a helpful naming convention](/docs/cli).
 
-Lastly, this is the bog standard CSS file. It contains the CSS classes that are used in the CSS Components.
+While this file uses CSS, you can use whatever styling solution you want. We'd recommend using SCSS; it works really well with CSS components!
 
 ```css
-.PanelWrapper {
+div.PanelWrapper {
```

Comment:
I fucking hate the div. convention John used. It unnecessarily increases CSS specificity, and then ties your class to a specific element you used, making it difficult to update. I'd prefer we don't recommend that, but up for debate

Reply:
will update the docs to mention that both options are available (in a separate PR)